java-frontend/src/main/java/org/sonar/java/bytecode/se/BytecodeEGWalker.java

```diff
@@ -748,11 +756,11 @@ private boolean handleMethodInvocation(Instruction instruction) {
       methodInvokedBehavior
         .exceptionalPathYields()
         .forEach(yield -> {
-          org.sonar.plugins.java.api.semantic.Type exceptionType = yield.exceptionType();
+          Type exceptionType = yield.exceptionType();
           yield.statesAfterInvocation(
             arguments, Collections.emptyList(), pop.state, () -> constraintManager.createExceptionalSymbolicValue(exceptionType)).forEach(ps -> {
-              ps.storeExitValue();
-              enqueueExceptionHandlers(exceptionType, ps);
+            ps.storeExitValue();
+            enqueueExceptionHandlers(exceptionType, ps);
           });
         });
       // FIXME : empty yields here should not happen, for now act as if behavior was not resolved.
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)